tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php

```diff
@@ -934,9 +936,32 @@ class A
 
 
 
-                    function A(){}}
+                        function A() {}
+                     }
                 ',
-                ['elements' => ['property']],
+                ['elements' => ['property' => ClassAttributesSeparationFixer::SPACING_ONE]],
+            ],
+            [
+                '<?php
+                    class A
+                    {
+                        private $a = null;
+                        public $b = 1;
+
+                        function A() {}
+                    }
+                ',
+                '<?php
+                    class A
+                    {
+                        private $a = null;
+
+                        public $b = 1;
+
+                        function A() {}
+                    }
+                ',
+                ['elements' => ['property' => ClassAttributesSeparationFixer::SPACING_NONE]],
```

Comment:
Please add more test cases using `SPACING_NONE` for methods and constants.

Reply:
Good call, found an issue https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4875/commits/117e617a6c0152622b2f2c427af5a157d14d4ec3

Do you think I'm missing any more tests?

Reply:
LGTM :+1: